core/build/src/lib.rs

```diff
@@ -66,7 +65,7 @@ pub fn sgx_builder() -> Builder {
         .derive_partialord(true)
         // Comments can cause doc tests to fail, see https://github.com/rust-lang/rust-bindgen/issues/1313
         .generate_comments(false)
-        .default_enum_style(EnumVariation::Consts)
+        .default_enum_style(EnumVariation::NewType { is_bitfield: false })
```

Comment:
Curious thoughts on this change
Consts style
```c
/* automatically generated by rust-bindgen 0.60.1 */

pub const SGX_QV_QVE_PATH: sgx_qv_path_type_t = 0;
pub const SGX_QV_QPL_PATH: sgx_qv_path_type_t = 1;
pub type sgx_qv_path_type_t = core::ffi::c_uint;
```

newstyle
```c
/* automatically generated by rust-bindgen 0.60.1 */

impl sgx_qv_path_type_t {
    pub const SGX_QV_QVE_PATH: sgx_qv_path_type_t = sgx_qv_path_type_t(0);
}
impl sgx_qv_path_type_t {
    pub const SGX_QV_QPL_PATH: sgx_qv_path_type_t = sgx_qv_path_type_t(1);
}
#[repr(transparent)]
#[derive(Debug, Copy, Clone, Hash, PartialOrd, Ord, PartialEq, Eq)]
pub struct sgx_qv_path_type_t(pub core::ffi::c_uint);  
```

Reply:
I think the style is fine, it just entails a bit more typing :-)